src/ControlKnob.vue

```diff
@@ -129,17 +129,28 @@ const mouseMoved = ref(false)
 const hasFocus = ref(false)
 const shiftModifier = ref(false)
 
-const downListener = (event: MouseEvent) => {
+const downListener = (event: MouseEvent | TouchEvent) => {
   mouseIsDown.value = true
   mouseMoved.value = false
-  prevY = event.clientY
-  preventScrolling(event)
+  prevY = getEventY(event);
+  preventScrolling(event);
+}
+
+ /** Gets the y coordinate associated with the event */
+function getEventY(event: TouchEvent | MouseEvent): number {
+  if (window.TouchEvent && event instanceof TouchEvent) {
+    return event.touches[0].pageY;
+  }
+  else if (event instanceof MouseEvent) {
+    return currentY = event.clientY
```

Comment:
Return shouldn't contain assignment.

Reply:
I will add parenthesis around the assignment to make the intention clear. If you really want to have a separate line, this is fine with me, too. Just update the PR. You should have write access anyway.

Reply:
I'd rather separate the assignment to another line. It's much more clear and explicit. I'm not pointing this out just to be nitpicky, it's just general good practice and makes the code more readable. See: https://eslint.org/docs/latest/rules/no-return-assign

The messiness of JS is one of my personal pet peeves about it as it usually leads to bugs, and in a FOSS project like this readability matters even more :)